### PR TITLE
Plugin compatibility with IDE 2022.3

### LINKED ISSUE #1466: Plugin is not compatible with IntelliJ IDE 2022.3
**Description**

UnitTestBot cannot be installed into the latest IDE: IDEA, PyCharm, WebStorm, because it has limitation up to 222.* version.
Latest PyCharm 2022.3 has 223.7571.203 version (came out on Dec 1)
Same for latest WebStorm 2022.3 (Nov 28)
Same for IDEA (Nov 30)

**To Reproduce**

1. [Download and install latest PyCharm Community](https://www.jetbrains.com/pycharm/download/#section=windows)
2. Download one of the latest plugin builds from main
3. File > Settings > Plugins > Install from disk > select downloaded UnitTestBot

**Expected behavior**

UnitTestBot is supposed to be installed.

**Actual behavior**

Error message that UnitTestBot is not compatible with this IDE version

**Visual proofs (screenshots, logs, images)**

image

`Plugin 'UnitTestBot' (version '2022.12.576.') is not compatible with the current version of the IDE, because it requires build 222.* or older but the current build is PC-223.7571.203`

**Environment**

Windows 10 Pro
IntelliJ PyCharm Community 2022.3 / IDEA / WebStorm

### PR DESCRIPTION
**Pull request by Vassiliy-Kudryashov**
# Description

**Note**: to be merged with low priority later
Updating versions and API compatibility fixes

Fixes #1466

## Type of Change

- Refactoring (typos and non-functional changes) 

# How Has This Been Tested?

## Manual Scenario 

runIde starts IDEA 223, the plugin should work by design

# Checklist (remove irrelevant options):

- [x] The change followed the style guidelines of the UTBot project
- [x] Self-review of the code is passed
- [x] No new warnings

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -40,7 +38,7 @@ collectionsVersion=0.3.4
 # utbot-intellij/build.gradle.kts
 # utbot-rd/build.gradle
 # utbot-rider/build.gradle.kts
-intellijPluginVersion=1.7.0
+intellijPluginVersion=1.11.0
```
Comment (Domonion) [inline gradle.properties]: ?

File: utbot-intellij/build.gradle.kts [hunk 2]
```diff
@@ -98,7 +95,7 @@ tasks {
 
     patchPluginXml {
         sinceBuild.set("212")
```
Comment (Domonion) [inline utbot-intellij/build.gradle.kts]: ?

File: utbot-ui-commons/build.gradle.kts [hunk 3]
```diff
@@ -43,7 +41,7 @@ tasks {
 
     patchPluginXml {
         sinceBuild.set("212")
-        untilBuild.set("222.*")
+        untilBuild.set("223.*")
```
Comment (Domonion) [inline utbot-ui-commons/build.gradle.kts:44]: remove

**Review by Domonion — APPROVED** [review]

**Comment by alisevych** [comment]
@Vassiliy-Kudryashov Could you please rename the PR before merging?
F.i.: Plugin compatibility with IDE 2022.3
And please remove bug # from the PR name. Fixes #bug in description is enough.

**Comment by Vassiliy-Kudryashov** [comment]
> @Vassiliy-Kudryashov Could you please rename the PR before merging? F.i.: Plugin compatibility with IDE 2022.3 And please remove bug # from the PR name. Fixes #bug in description is enough.

Done!

**Pull request merged**